sdk/java-sdk-spring/src/main/scala/kalix/spring/impl/RestKalixClientImpl.scala

```diff
@@ -16,12 +16,15 @@
 
 package kalix.spring.impl
```

Comment:
Or ready for review, I simplified the old code a bit and add separate methods for the typed client.